### DISCUSSION TITLE
Set Hotend Fan Speed

### DISCUSSION THREAD

**Discussion by PhoenixG36** [Jyers/Marlin#832] [Q&A] [answered]
Hi, I use an Enders V2 printer and replaced the hotend fan with an sunon.
Now I'm want to set the speed from 255 to 253. (much more silend)
Is the an option to do this?

What I found by searching didn´t change the speed of this fan :-(

https://forum.prusaprinters.org/forum/hardware-firmware-and-software-help/hotend-fan-speed-how-to-change/

Thx for helping.

**Comment by Reggi3** [comment, marked as answer]
There are 2 fans in that area, the hotend fan and the part cooling fan, the hotend fan is always on and you cannot control it through marlin.

**Reply by PhoenixG36** [reply]
Is there no way to change the setting and compile the firmware new?

**Reply by feelfree69** [reply]
No, you would have to tweak the hardware, eg. by adding a resistor.
